h2/src/main/org/h2/expression/function/Function.java

```diff
@@ -2291,6 +2300,41 @@ private static Value oraHash(Value value, long bucket, long seed) {
         return ValueBigint.get((hc & Long.MAX_VALUE) % (bucket + 1));
     }
 
+    private static Value base64encode(byte[] data, String algorithm) {
+        if (data!=null){
```

Comment:
`data` is not expected to be `null` here, if it can, something else is going wrong. If function does not declare `NULL` support, `getValueWithArgs()` returns `NULL` if any of arguments is `NULL` by itself at the beginning of the method.